controller/array_action/array_mediator_svc.py

```diff
@@ -783,8 +784,11 @@ def _get_array_iqns_by_node_id(self):
         logger.debug("Found iqns by node id: {}".format(array_iqns_by_id))
         return array_iqns_by_id
 
-    def _list_ip_ports(self):
+    def _list_ip_ports(self, port_set_id):
```

Comment:
consider using the same name as in the storage, i.e. `portset` instead of `port_set`, similar to how we did with `fcmap` (i.e. not `fc_map`)